scripts/maintenance/migrate_project/src/models.py

```diff
@@ -0,0 +1,86 @@
+import json
+from pathlib import Path
+from typing import Optional
+from uuid import UUID
+
+from pydantic import BaseModel, Field
+from sqlalchemy import desc
+
+
+class DBConfig(BaseModel):
+    address: str
+    user: str
+    password: str
+    database: str
+
+
+class S3Config(BaseModel):
+    endpoint: Optional[str] = None
+    access_key: str
+    secret_key: str
+    bucket: str
+
+
+class SourceConfig(BaseModel):
+    db: DBConfig
+    s3: S3Config
+    project_uuid: UUID = Field(..., description="project to be moved from the source")
+    hidden_projects_for_user: Optional[int] = Field(
+        None,
+        description="by default nothing is moved, must provide an user ID for which to move the hidden projects",
+    )
+
+
+class DestinationConfig(BaseModel):
+    db: DBConfig
+    s3: S3Config
+    user_id: int = Field(
+        ...,
+        description="new projects owner under which to save files and projects",
+    )
+    user_gid: int = Field(
+        ...,
+        description="group id for the user, required to give the correct permissions to the user",
+    )
+
+
+class Settings(BaseModel):
```

Comment:
Why not ``BaseSettings``?

Reply:
It was born as an interdependent script, in the end I had to add the `postgres-database`, that's why no dependency on BaseSettings. Also I wanted to have all configuration in a file. I think it is simpler to run like that.

Reply:
BaseSettings is part of pydantic... I did not mean our BaseCustomeSettings